From f51ac206b1ca27d240fe7fde6975496d558f51df Mon Sep 17 00:00:00 2001
From: Luka Perkov <luka@openwrt.org>
Date: Sun, 4 Aug 2013 03:10:59 +0200
Subject: [PATCH] usb: chipidea: enable to build host support as module

USB_CHIPIDEA_HOST does not need to depend on USB=y, USB_CHIPIDEA_HOST will work
just fine even if USB=m is used. The depends line can be safely removed since
USB_CHIPIDEA already depends on USB.

Tested on Gateworks imx6q Ventana board (gw-5400-a) and imx6dl Wandboard Dual
(imx6dl-wandboard).

Signed-off-by: Luka Perkov <luka@openwrt.org>
---
 drivers/usb/chipidea/Kconfig | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/chipidea/Kconfig
+++ b/drivers/usb/chipidea/Kconfig
@@ -19,7 +19,6 @@ config USB_CHIPIDEA_UDC
 
 config USB_CHIPIDEA_HOST
 	bool "ChipIdea host controller"
-	depends on USB=y
 	depends on USB_EHCI_HCD=y || (USB_CHIPIDEA=m && USB_EHCI_HCD=m)
 	select USB_EHCI_ROOT_HUB_TT
 	help
